modules/ROOT/pages/installing-mobile-services.adoc

```diff
@@ -10,16 +10,139 @@ NOTE: {org-name} recommends running OpenShift using the method described here. H
 
 * MacOS or Linux
 
-* Docker
+* link:https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html[Ansible] version 2.3 or later
 
-. Install OpenShift using the instructions at https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md
+* link:https://www.openshift.org/download.html[Openshift client tools] version 3.9 or later
 
+* Docker must be version 17.09.1 in order to be compatible with the latest release of OpenShift and depending on your OS this is downloaded from different places:
 
+[role="primary"]
+.MacOS
 
-== Next Steps
+****
+You can download Docker CE for MacOS version 17.09.1 link:https://download.docker.com/mac/stable/19543/Docker.dmg[here].
+****
+
+[role="secondary"]
+.Linux
+
+****
+You can download Docker CE for Linux version 17.09.1 link:https://download.docker.com/linux/centos/7/x86_64/stable/Packages/docker-ce-17.09.1.ce-1.el7.centos.x86_64.rpm[here].
+****
+
+== Local development
+
+. *Mobile-core Installer*
++
+We configure the local development installation of {org-name} {product-name} using ansible scripts in our link:https://github.com/aerogear/mobile-core[mobile-core] repo. Clone this repo to your local machine and check out the {release-number} tag using:
++
+[source,bash,subs="attributes"]
+----
+git clone git@github.com:aerogear/mobile-core.git
+git checkout {release-number}
+----
 
-* Provision a Mobile Service
+. *Installation*
++
+The installation process is straightforward when using the `install.sh`
+script - which is available in the mobile-core repo we have already
+cloned in the requirements section - with the following command:
++
+[source,bash]
+----
+./installer/install.sh
+----
++
+This validates that Ansible, Docker and the OpenShift Client Tools
+are installed and are the right version.
++
+Once verified it prompts you for your Docker Hub login credentials, which
+must be valid for the installation to continue.
++
+The next set of values are optional and can be left with the default values:
++
+[source,bash]
+----
+DockerHub Tag (Defaults to {release-number}):
```

Comment:
@finp `{release-number}` here is rendered as is in the code block. String interpolation doesn't work.